packages/panzer/mini-em/src/responses/MiniEM_AddFieldsToMesh.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef _MiniEM_AddFieldsToMesh_hpp_
+#define _MiniEM_AddFieldsToMesh_hpp_
+
+#include "Teuchos_ParameterList.hpp"
+#include "Panzer_STK_Interface.hpp"
+
+namespace mini_em {
+
+  void addFieldsToMesh(panzer_stk::STK_Interface & mesh,
```

Comment:
You don't have to include `Panzer_STK_Interface.hpp` here.  Since you only refer to it by reference, you may use a forward declaration instead.

Reply:
Ditto for `Teuchos_ParameterList.hpp` (which includes a HUGE number of header files).